docker-compose.yml

```diff
@@ -74,19 +78,50 @@ services:
     environment:
       - POSTGRES_USER=admin
       - POSTGRES_PASSWORD=admin_password
-      - DATABASE_NAME=court-register-db
-      - DATABASE_ENDPOINT=court-register-db
+      - POSTGRES_DB=court_register_db
+
+  prison-register:
+    image: quay.io/hmpps/prison-register:latest
+    networks:
+      - hmpps
+    container_name: prison-register
+    depends_on:
+      - hmpps-auth
+    ports:
+      - "8103:8080"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8080/health/ping"]
+    environment:
+      - SERVER_PORT=8080
+      - SPRING_PROFILES_ACTIVE=dev
```

Comment:
Using `dev` profile which spins up in memory H2 db, it didn't seem necessary to instantiate another Postgres instance like Court Register uses